### PR TITLE
[WIP] British English

### PR DESCRIPTION
**Pull request by jamierocks**
So far I have done very little conversion, other contributors can hopefully help towards completing this ;)

### CODE REVIEW

**Comment by cb88** [comment]
How is this different from the English that is already supported? I mean, it's not like it's written in Redneck...  https://www.google.com/search?q=redhat+redneck&tbm=isch

File: conf/locale/locale_en_GB.ini [hunk 1]
```diff
@@ -0,0 +1,1060 @@
...
+
+repository = Repository
+organization = Organization
+mirror = Mirror
+new_repo = New Repository
+new_migrate = New Migration
+new_mirror = New Mirror
+new_fork = New Fork Repository
+new_org = New Organization
+manage_org = Manage Organizations
```
Comment (thehowl) [inline conf/locale/locale_en_GB.ini]: \>british english 
\>organizations 
\>organi**z**ations

**Comment by jamierocks** [comment]
Many common differences between American English and British English include:
-our -> -or eg colour -> color
There are other things like:
lift -> elevator

And generally British English uses -ise, not -ize

File: conf/locale/locale_en_GB.ini (hunk 1 above)
Comment (jamierocks) [inline conf/locale/locale_en_GB.ini]: I never claimed for this to be finished.

**Comment by Unknwon** [comment]
Thanks your PR!

However, please this is [not going to be merged](https://github.com/gogits/gogs/wiki/Contributing-Code#those-we-do-not-accept).

If you wish, I can add target language on Crowdin for you.

**Comment by jamierocks** [comment]
Ah, that'd be awesome if you could add it as a target language. ;)

**Comment by Unknwon** [comment]
Here you go: https://crowdin.com/translate/gogs/1/enus-engb :trollface:

**Comment by Unknwon** [comment]
BTW, you can send a copy of locale_en-GB to u@gogs.io so I can upload strings that are already translated for you.

**Pull request closed without merging**